plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -320,6 +326,36 @@ func (c *nodePlugin) admitPVCStatus(nodeName string, a admission.Attributes) err
 	}
 }
 
+// diffsAreWhitelistedTaints returns true if and only if the difference between
+// oldTaints and newTaints are whitelisted taints only.
+func diffsAreWhitelistedTaints(newTaints []api.Taint, oldTaints []api.Taint) bool {
```

Comment:
I made this function a bit more generic and not specific to "NotReady" taint, although it still effectively checks "NotReady" which is the only whitelisted taint.